src/babashka/nrepl/impl/utils.clj

```diff
@@ -7,14 +7,21 @@
 
 (set! *warn-on-reflection* true)
 
+;; TODO: should be based on this method here https://github.com/nrepl/nrepl/blob/0895caa7f0910c013014d554bbb262834a8408e4/src/clojure/nrepl/middleware/print.clj#L122
```

Comment:
@borkdude I feel I have to add this particular method from `nrepl.middleware.pprint`. Is my understanding correct?

Reply:
Not necessarily. The method in nREPL only exists as some default/example that people can use out-of-the-box for pretty-printing.